src/bindings/python/flux/job/Jobspec.py

```diff
@@ -18,6 +18,7 @@
 
 import yaml
 from _flux._core import ffi
+from flux import hostlist, idset
```

Comment:
commit message typo, "validation of hostlist and rank constraints", plural ranks?